parachains/runtimes/assets/statemine/src/xcm_config.rs

```diff
@@ -251,11 +255,20 @@ pub type Barrier = DenyThenTry<
 		TakeWeightCredit,
 		AllowTopLevelPaidExecutionFrom<Everything>,
```

Comment:
Does not technically need to be under `WithComputedOrigin` owing to the permissive `Everything` origin filter, however if this ever got changed to something less permissive then it probably would need to be moved under `WithComputedOrigin`, therefore I think it would be best to put it there now to prevent potential mishap later. (Same with the other chains.)